kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js

```diff
@@ -146,6 +146,8 @@ onEvent('recipes', (event) => {
         'simplefarming:apple_pie',
         'simplefarming:blueberry_pie',
 
+        'tconstruct:tools/modifiers/wither_bone_conversion',
```

Comment:
What does removing this effectively do? 😄

Reply:
It's a conflict. Shapeless one wither bone to one normal bone. But with the tags it can be converted to bone meal already with that recipe.

If we really want a conversion to normal bone, we can add a shaped craft, I guess. Just didn't seem that important.

Reply:
Oh alright, because of the name I thought it was the "recipe" for the Necrotic Bone modifier.